launcher/modplatform/modrinth/ModrinthAPI.h

```diff
@@ -101,9 +96,9 @@ class ModrinthAPI : public NetworkModAPI {
         return s.isEmpty() ? QString() : QString("[%1],").arg(s);
     }
 
-    inline auto validateModLoader(ModLoaderType modLoader) const -> bool
+    inline auto validateModLoaders(ModLoaderTypes loaders) const -> bool
     {
-        return modLoader == Unspecified || modLoader == Forge || modLoader == Fabric || modLoader == Quilt;
+        return loaders == Unspecified || loaders & (Forge | Fabric | Quilt);
```

Comment:
```suggestion
        return ((loaders == Unspecified) || (loaders & (Forge | Fabric | Quilt)));
```

This one may seem a bit of overkill, so it's fine if this is left as-is :)

Reply:
Maybe like this:
```suggestion
        return (loaders == Unspecified) || (loaders & (Forge | Fabric | Quilt));
```
Slightly less overkill